manifests/init.pp

```diff
@@ -205,6 +205,8 @@
   Integer[1,65536]                                     $redis_server_port,
   Integer[1]                                           $redis_timeout,
   Stdlib::Absolutepath                                 $unbound_conf_d,
+  Boolean                                              $manage_roothints_file,
+  Optional[String]                                     $hints_file_content,
```

Comment:
I think empty string isn't allowed here/doesn't make sense? In that case we can update it to this:
```suggestion
  Optional[String[1]]                                  $hints_file_content,
```